src/Tribe/Tickets_Handler.php

```diff
@@ -880,9 +880,13 @@ public function get_ticket_totals( $ticket ) {
 
 		$totals = array_map( 'intval', $totals );
 
-		// Remove Pending from total
+		// Remove Pending from total.
 		$totals['sold'] -= $totals['pending'];
 
+		// Update Stock.
+		$totals['stock'] = tribe_tickets_get_capacity( $ticket->ID ) - $totals['sold'];
+		update_post_meta( $ticket->ID, '_stock', $totals['stock'] );
```

Comment:
Can we find a more appropriate place for such an operation ? Like when the attendee is moved. I would not expect any write operations to be part of a method `get_ticket_totals`

Reply:
During the move there are are a couple of update posts, that are wrong because this function returns wrong totals. Have [a look here](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tribe/Admin/Move_Tickets.php#L637)

Reply:
This placement is definitely problematic @pattihis. It can very easily cause infinite loops. see method `migrate_object_capacity` which can be called by `tribe_tickets_get_capacity` which in turn calls `get_ticket_totals`. I would look for another place for such an operation during a different hook

Reply:
If you check CI tests are failing because of infinite loop as well currently